test/Npgsql.OpenTelemetry.Tests/NpgsqlTracingOptionsTests.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using Npgsql.Tests;
+using NUnit.Framework;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+namespace Npgsql.OpenTelemetry.Tests;
+
+[NonParallelizable]
+public class NpgsqlTracingOptionsTests : TestBase
```

Comment:
Thanks for adding test coverage... I've opened #4285 to track doing this for the entire tracing functionality.